src/game/entities/Cup.js

```diff
@@ -0,0 +1,36 @@
+import Phaser from 'phaser';
+
+export default class Cup extends Phaser.Physics.Matter.Sprite {
```

Comment:
For now it's not a Cup, but a CupPart (since it's used to create a full fledged Cup in Game).

The whole cup should be created in here and then added to the Game as a single object.